Formula/bareos-client.rb

```diff
@@ -21,29 +21,30 @@ class BareosClient < Formula
   depends_on "openssl@1.1"
   depends_on "readline"
 
-  conflicts_with "bacula-fd",
-    because: "both install a `bconsole` executable"
+  conflicts_with "bacula-fd", because: "both install a `bconsole` executable"
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args,
-                            "-Dworkingdir=#{var}/lib/bareos",
-                            "-Darchivedir=#{var}/bareos",
-                            "-Dconfdir=#{etc}/bareos",
-                            "-Dconfigtemplatedir=#{lib}/bareos/defaultconfigs",
-                            "-Dscriptdir=#{lib}/bareos/scripts",
-                            "-Dplugindir=#{lib}/bareos/plugins",
-                            "-Dfd-password=XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX",
-                            "-Dmon-fd-password=XXX_REPLACE_WITH_CLIENT_MONITOR_PASSWORD_XXX",
-                            "-Dbasename=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dhostname=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
-                            "-Dclient-only=ON"
-      system "make"
-      system "make", "install"
-    end
+    args = %W[
+      -DENABLE_PYTHON=OFF
```

Comment:
No idea how to only disable Python 2.

`bareos-client` uses `find_package(Python2 ...)` and `find_package(Python3 ...)`. - https://github.com/bareos/bareos/blob/Release/21.1.2/core/cmake/BareosFindAllLibraries.cmake#L45-L46

Reply:
Could we just `inreplace "Python2", "Python3"`?

Reply:
Even manually disabling Python 2 causes other failures with Python 3 as it tries to install into Python formula Framework directory. For now, will drop Python plugins.